python/lsst/ts/xml/data/sal_interfaces/Reflector/Reflector_Commands.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>Reflector</Subsystem>
+    <EFDB_Topic>Reflector_command_switchOn</EFDB_Topic>
+    <Description>Switch on reflector.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>Reflector</Subsystem>
+    <EFDB_Topic>Reflector_command_switchOff</EFDB_Topic>
+    <Description>Switch off reflector.</Description>
+  </SALCommand>
+</SALCommandSet>
```

Comment:
```suggestion
  <SALCommand>
    <Subsystem>Reflector</Subsystem>
    <EFDB_Topic>Reflector_command_switch</EFDB_Topic>
    <Description>Switch on/off reflector.</Description>
    <item>
    	<EFDB_Name>toggle</EFDB_Name>
    	<Description>Switch on/off the reflector</Description>
    	<IDL_Type>bool</IDL_Type>
    	<Count>1</Count>
    </item>
  </SALCommand>
</SALCommandSet>
```
I think this could just be one command with a boolean toggle value for switching on the reflector and and switching off the reflector.